### ISSUE TITLE
The model trained on the official dataset without pre-training yields all zeros on the validation set.

### ISSUE THREAD

**Issue by zhibeiyou135** [SJTU-LuHe/TransVOD#43]
What could be the issue here
<img width="940" alt="Snipaste_2024-08-02_14-28-47" src="https://github.com/user-attachments/assets/04754fc1-5f4e-4657-81b4-9b95a5f9bd2f">
